fiftyone/core/collections.py

```diff
@@ -10548,42 +10523,132 @@ def create_index(self, field_or_spec, unique=False, wait=True, **kwargs):
                 "or all frame-level fields"
             )
 
+        to_db_name = lambda spec: "_".join("%s_%s" % (f, o) for f, o in spec)
```

Comment:
Unlikely that people are using custom names, but just want to call out that this assumes the default naming scheme and wouldn’t work for named indexes